src/transformState.js

```diff
@@ -1,11 +1,34 @@
-'use strict';
+"use strict";
 
 /**
  * @param {Object} state
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const obj of actions) {
```

Comment:
Destructure values from `action`
```suggestion
  for (const action of actions) {
```